src/App.tsx

```diff
@@ -1,56 +1,90 @@
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useRef, useState,
+} from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
for fix such error issue commit and push changes in package.json file. in general better to commit and push changes in all files .
![Screenshot from 2023-12-09 15-45-16](https://github.com/mate-academy/react_autocomplete/assets/33029864/c1ed24b7-6ade-47d8-950b-75e234dee4e4)